cookbook/core/containerization/image_sepc_example.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
Can you fix the name of the file: sepc to spec?

Reply:
oops, thanks